src/main/java/io/r2dbc/postgresql/message/Format.java

```diff
@@ -49,10 +47,9 @@
      * @return the enum constant of this type with the specified discriminator
      */
     public static Format valueOf(short s) {
-        return Arrays.stream(values())
-            .filter(type -> type.discriminator == s)
-            .findFirst()
-            .orElseThrow(() -> new IllegalArgumentException(String.format("%d is not a valid format", s)));
+        if ( s == BINARY.discriminator ) return BINARY;
```

Comment:
Curly braces here as well please.

Reply:
Actually, scratch the previous. Can you please match this to the style you used in `BackendMessageDecoder` instead?

Reply:
given there are only 2 the loop seems redundant ? I'd be surprised if the compiler didn't optimize it out.

Reply:
My main concern is consistency here.  I'd like people to look into any enumeration and see code that they're familiar with.  I'm with you that the compiler likely optimizes this out, and given that, I'd like to use the more generic form so that it stretches across all implementations.